pkgs/development/python-modules/seahub/default.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenv, lib, fetchFromGitHub, python3Packages, makeWrapper }:
+
+python3Packages.buildPythonPackage rec {
```

Comment:
Is this supposed to be used as `import seahub`? If not, this should not be in `pythonPackages` and should use `buildPythonApplication`.